pydata_sphinx_theme/_templates/last-updated.html

```diff
@@ -0,0 +1,3 @@
+<p class="last-updated">
```

Comment:
The original had a `if last_updated`, should that be kept?

Reply:
I was thinking that people could just provide this in the footer config if they wished, rather than using the Sphinx feature flag. Does that make sense?